### DISCUSSION TITLE
OT Deployment

### DISCUSSION THREAD

**Discussion by baclaeys** [amidaware/tacticalrmm#950] [Q&A]
I am interested in deploying this software in an OT environment where internet access is limited.  Is there a way to bypass the A records and the Let's Encrypt certificates?

**Comment by wh1te909** [comment]
you don't external A records, just need to resolve dns internally either through hosts files or local dns server
for let's encrypt you can generate the wildcard certificate on any host that has internet and then just copy the cert to the machine hosting your rmm

**Reply by silversword411** [reply]
In an air gapped network, how do you handle cert chain validation?
